pyani_plus/methods/method_anib.py

```diff
@@ -79,7 +79,7 @@ def fragment_fasta_file(
         raise ValueError(msg)
 
 
-def parse_blastn_file(blastn: Path) -> tuple[float, int, int]:
+def parse_blastn_file(blastn: Path) -> tuple[float | None, int | None, int | None]:
```

Comment:
Would it be more easier to read if we change it to:

```python
def parse_blastn_file(blastn: Path) -> tuple[float, int, int] | tuple[None, None, None]:
```

Reply:
Easier to read, yes - and I tried that initially.

However, ``mpypy`` complains. I think it _should_ be possible with some refactoring to make it explicit in the code that you either get numerical values, or all None.